src/index.d.ts

```diff
@@ -673,6 +673,7 @@ export interface SupportLanguage {
   linguistLanguageId?: number | undefined;
   vscodeLanguageIds?: string[] | undefined;
   interpreters?: string[] | undefined;
+  isSupported?: (file: string) => boolean | undefined;
```

Comment:
I don't think we need `| undefined` here.

Reply:
Like above, and we're using `??` check for `fileInfo.physicalFile` vs `fileInfo.file`.

Reply:
We can accept any truthy/falsy value internally, but I don't like it explicitly documented.

Reply:
Oh, maybe I was going to write `isSupported?: ((file: string) => boolean) | undefined;` actually. 😂